### PR TITLE
[CI-4986] Golang change to enable docker runner initialization step

### PR DESCRIPTION
**Pull request by happyhebaby**

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -28,6 +28,14 @@ type (
 		MountDockerSocket *bool             `json:"mount_docker_socket,omitempty"`
 	}
 
+	SetupRequest struct {
```
Comment (shubham149) [inline api/api.go:31]: Why do we require changes to this api?

File: config/config.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ type Config struct {
 	}
 
 	Client struct {
-		Bind       string `envconfig:"HTTPS_BIND" default:":9079"`
+		Bind       string `envconfig:"HTTPS_BIND" default:":3000"`
```
Comment (eoinmcafee00) [inline config/config.go:27]: Why is this changed? This will break everything.

**Review by tphoney — CHANGES_REQUESTED** [review]
This cannot be merged until we have the unit tests running
